src/map/mob.c

```diff
@@ -4630,13 +4866,10 @@ void mob_load(bool minimal) {
 	}
 	sv->readdb(map->db_path, "mob_item_ratio.txt", ',', 2, 2+MAX_ITEMRATIO_MOBS, -1, mob->readdb_itemratio); // must be read before mobdb
 	mob->readchatdb();
-	if (map->db_use_sql_mob_db) {
```

Comment:
This removes the SQL mob database, without removing it entirely (config files, config parser). I have a commit in the `deprecationwarning` branch that gets rid of teh config files entries as well, I shall merge that before merging this pull request